actionmailbox/app/controllers/action_mailbox/ingresses/amazon/inbound_emails_controller.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+module ActionMailbox
+  # Ingests inbound emails from Amazon SES/SNS and confirms subscriptions.
+  #
+  # Subscription requests must provide the following parameters in a JSON body:
+  # - +Message+: Notification content
+  # - +MessagId+: Notification unique identifier
+  # - +Timestamp+: iso8601 timestamp
+  # - +TopicArn+: Topic identifier
+  # - +Type+: Type of event ("Subscription")
+  #
+  # Inbound email events must provide the following parameters in a JSON body:
+  # - +Message+: Notification content
+  # - +MessagId+: Notification unique identifier
+  # - +Timestamp+: iso8601 timestamp
+  # - +SubscribeURL+: Topic identifier
+  # - +TopicArn+: Topic identifier
+  # - +Type+: Type of event ("SubscriptionConfirmation")
+  #
+  # All requests are authenticated by validating the provided AWS signature.
+  #
+  # Returns:
+  #
+  # - <tt>204 No Content</tt> if a request is successfully processed
+  # - <tt>401 Unauthorized</tt> if a request does not contain a valid signature
+  # - <tt>404 Not Found</tt> if the Amazon ingress has not been configured
+  # - <tt>422 Unprocessable Entity</tt> if a request provides invalid parameters
+  #
+  # == Usage
+  #
+  # 1. Install the {aws-sdk-sns}[https://rubygems.org/gems/aws-sdk-sns] gem:
+  #
+  #        # Gemfile
+  #        gem "aws-sdk-sns", "~> 1.9", require: false
+  #
+  # 2. Tell Action Mailbox to accept emails from Amazon SES:
+  #
+  #        # config/environments/production.rb
+  #        config.action_mailbox.ingress = :amazon
+  #
+  # 3. Configure which SNS topics will be accepted:
+  #
+  #        config.action_mailbox.amazon.subscribed_topics = %w(
+  #          arn:aws:sns:eu-west-1:123456789001:example-topic-1
+  #          arn:aws:sns:us-east-1:123456789002:example-topic-2
+  #        )
```

Comment:
Will someone going through these steps know what SNS topics to accept before step 4?